### DISCUSSION TITLE
Authentication parameters of the token strategy

### DISCUSSION THREAD

**Discussion by hongzhouzi** [kiali/kiali#5892] [Q&A] [answered]
kiali configured token strategy, in client request graph data should be how to pass authentication parameters?
Adding the Authorization header does not work.

kiali version: 1.50.1

**Comment by jmazzitelli** [comment, marked as answer]
_converting to a Q&A discussion_

Docs about the token auth strategy are here: https://kiali.io/docs/configuration/authentication/token/ ... as the docs state, it is similar to the k8s login mechanism as described here: https://github.com/kubernetes/dashboard/blob/master/docs/user/access-control/README.md#login-view

If you want to pass your auth information via the Authentication header, you need to use the `header` auth strategy. That is documented here: https://kiali.io/docs/configuration/authentication/header/

You can see how the integration tests test these two auth strategies to get a feel for how its done programatically:

* https://github.com/kiali/kiali-operator/blob/v1.64.0/molecule/asserts/token-test/assert-token-access.yml
* https://github.com/kiali/kiali-operator/blob/v1.64.0/molecule/header-auth-test/converge.yml#L157-L158
